polaris-react/src/components/Button/Button.scss

```diff
@@ -874,6 +876,13 @@
   &.sizeLarge {
     padding-left: var(--p-space-3);
     padding-right: var(--p-space-3);
+
+    #{$se23} & {
+      // stylelint-disable -- polaris/conventions/polaris/custom-property-allowed-list
```

Comment:
Heads up, this is disabling all Stylelint rules (text after `--` is the description). Perhaps, this is intentional though since you seem to be disabling specific rules "correctly" elsewhere.